docs/execution-providers/NNAPI-ExecutionProvider.md

```diff
@@ -22,7 +22,9 @@ Accelerate ONNX models on Android devices with ONNX Runtime and the NNAPI execut
 The NNAPI Execution Provider (EP) requires Android devices with Android 8.1 or higher. It is recommended to use Android devices with Android 9 or higher to achieve optimal performance.
 
 ## Install
-Pre-built packages of ONNX Runtime Mobile with NNAPI EP for Android are published on Maven: [com.microsoft.onnxruntime:onnxruntime-mobile](https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-mobile).
+Pre-built packages of ONNX Runtime with NNAPI EP for Android are published on Maven.
+
+See [here](../install/index.md#install-on-android) for installation instructions.
 
```

Comment:
We're removing the link to Maven but I didn't see any other place where the user is provided that info. Can we add 2 bullet points - one with the link to maven for the mobile package and one with the link to the full package?